src/solvers/dgsem_tree/indicators_2d.jl

```diff
@@ -125,6 +125,292 @@ function (indicator_hg::IndicatorHennemannGassner)(u::AbstractArray{<:Any,4},
 end
 
 
+function (indicator_hg::IndicatorHennemannGassner)(u::AbstractArray{<:Any,4}, mesh::Union{UnstructuredMesh2D},
```

Comment:
If this is really specific to the `UnstructuredMesh2D`, it shouldn't life in this folder but in the appropriate one.